ext/vscode/package.nls.json

```diff
@@ -17,6 +17,7 @@
 
     "azure-dev.commands.cli.deploy.title": "deploy (deploy the application's code to Azure)",
     "azure-dev.commands.cli.restore.title": "restore (restores application dependencies)",
+    "azure-dev.commands.cli.package.title": "package (packages application source and dependencies)",
```

Comment:
/cc @savannahostrowski Do you like this string?

Reply:
LGTM!